lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1224,70 +1224,349 @@
   ],
   "ImageElements": [
     {
-      "src": "http://localhost:10200/dobetterweb/lighthouse-480x318.jpg",
-      "displayedWidth": 480,
-      "displayedHeight": 57,
+      "src": "http://localhost:59523/dobetterweb/lighthouse-480x318.jpg?iar1",
```

Comment:
hmm, how did the port change? did you use `yarn update:sample-artifacts ImageElements` to update this?

Reply:
Yeah i did use that 👀

Reply:
it's b/c of this line https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/scripts/update-report-fixtures.js#L22 . I expected it to use the same port as the static server (10200). it being random is not good (code churn, if a line changes in a commit it should be because it really changed, not because there's a random number)

Altho this script has been like this since it was introduced 3 years ago #4793 ...

these are all the ports I found in artifacts.json

```
localhost:10200
localhost:50114
localhost:53133
localhost:54759
localhost:54819
localhost:9876
```

Eh, I'll file a bug but since this has existed forever this PR doesn't have to worry about it.